src/trajectories/transition.jl

```diff
@@ -1,9 +1,10 @@
-struct TransitionPathEnsemble
-    paths::Vector
-    times::Vector
-    success_rate::Real
-    t_res::Real
-    t_trans::Real
+struct TransitionPathEnsemble{SSS,T,Tstat,ES}
```

Comment:
We could rename this to `TransitionEnsemble` as you suggested, it's indeed shorter

Reply:
Yes I agree